website/docs/getting_started.mdx

```diff
@@ -80,16 +81,24 @@ Then install packages with `flutter pub get`.
   {...dartPubspec}
 ></AutoSnippet>
 
-Then install packages with `dart pub get`.
+Then, install packages with `dart pub get`.
 
 <ConditionalSnippet codegen={true}>
-  Finally, run the code-generator with{" "}
-  <code>flutter pub run build_runner watch</code>
+  Run the code-generator with{" "}
+  <code>flutter pub run build_runner watch</code>.
 </ConditionalSnippet>
 
 </TabItem>
 </Tabs>
 
+Enable `custom_lint`'s plugin in your `analysis_options.yaml`:
+
+<AutoSnippet
```

Comment:
You don't need an "AutoSnippet" here.

This component is something I made for when the content needs to change if codegen/hooks are enabled/disabled. But that's not the case here

Reply:
Oh, perfect! I'm glad to know it's much easier than I thought.

Reply:
Okay, take a look now. I used a `<CodeBlock>` instead.